libsplinter/src/admin/client/event/ws/actix_web_client.rs

```diff
@@ -415,10 +417,25 @@ impl From<messages::SplinterNode> for CircuitMembers {
         Self {
             node_id: splinter_node.node_id,
             endpoints: splinter_node.endpoints,
+            #[cfg(feature = "challenge-authorization")]
+            public_key: splinter_node
+                .public_key
+                .as_ref()
+                .map(|public_key| to_hex(&public_key)),
         }
     }
 }
 
+#[cfg(feature = "challenge-authorization")]
+fn to_hex(bytes: &[u8]) -> String {
+    let mut buf = String::new();
+    for b in bytes {
+        write!(&mut buf, "{:02x}", b).expect("Unable to write to string");
```

Comment:
panic should not occur within the library

Reply:
Updated to use the implementation in the library at hex.rs.

Reply:
This shoud have been used anyway as it is already pulled into the file.